HTCONDOR-2848 document dollar dollar $$ in container_image (#3110)
htcondor/htcondor: docs/users-manual/env-of-job.rst

will parse this image and advertise what type of container image it
is, and match with startds that can support that image.

> Note that the container image, like most other submit file commands,
> contain a $$ expansion.  This may be useful if you would like to use 

Reviewer: ```suggestion
Note that :subcom:`container_image`, like most other submit file commands,
can contain a ``$$`` expansion.  This may be useful if you would like to use 
```
